app/controllers/metrics_controller.rb

```diff
@@ -51,8 +41,6 @@ def update
     end
   end
 
-  # DELETE /metrics/1
-  # DELETE /metrics/1.json
   def destroy
     @metric.destroy
     respond_to do |format|
```

Comment:
@eliotsykes I am not sure. Part of me likes the idea of letting anybody edit the metrics. If the person who added them leaves Zendesk, it would be nice to let the person who takes over management of the feature edit the orphaned content. At this point it seems easier to me to leave everything open to any registered user. What do you think?